examples/getting_started/benchmarking_config_titanic.yaml

```diff
@@ -0,0 +1,11 @@
+experiment_name: titanic_gbm_benchmarking_run
```

Comment:
This is a bit overhwelming to put in getting started. I would move it to a `benchmarking` directory in the examples folder.